UtopiaCity/Controllers/Timeline/TimelineBasicController.cs

```diff
@@ -123,5 +132,21 @@ public async Task<IActionResult> Details(string id)
             return View("DetailsTimelineView", chosenEvent);
         }
 
+        //VIEW DATA 
+        public ActionResult ViewModelResult()
+        {
+            CollectionDataModel dataViewModel = new CollectionDataModel();
+            
+            List<Flight> flights = _flightService.GetFlightList();
+            
+            List<Event> events = _lifeService.GetAll();
+            
```

Comment:
var?

Reply:
I did not think about it. Why not)

Reply:
However, I leave a longer value List<Flights> flights. Refactoring will be later

Reply:
ok